incubator/artifactory/templates/NOTES.txt

```diff
@@ -0,0 +1,6 @@
+Artifactory can be accessed via port 8081 on the following DNS name from within your cluster:
+{{ template "fullname" . }}.{{ .Release.Namespace }}.svc.cluster.local
+
```

Comment:
Add examples of how to connect to this from outside the cluster. Check out the jenkins chart for an example of how to differentiate between the various service modes.